src/icons/annotations/index.md

```diff
@@ -0,0 +1,26 @@
+**Annotation icons**
```

Comment:
Does Github display `index.md` files when you navigate to the folder via a browser? If not, could we name these centralized files `README.md`?

Reply:
Nope, doesn't seem like it. I switched them to `README.md`.